packages/gatsby-plugin-google-tagmanager/src/gatsby-ssr.js

```diff
@@ -19,6 +19,20 @@ exports.onRenderBody = (
         : ``
 
     setHeadComponents([
+      ...(pluginOptions.dataLayerName
```

Comment:
We shouldn't need to do this right?

Reply:
You are correct that it is not required for renaming the dataLayer.  The reason I included this was because of my use case where my embedded React app, which used a different GTM container with the renamed dataLayer, needed to be pushed to the second GTM container configured in Gatsby (which used the default name dataLayer).  However, because most users of the plugin will only have one GTM container and will not encounter this use case, it should be removed.  I'll take this part out now and will update the changes.  Thanks for your help.